### DISCUSSION TITLE
How to sort by column

### DISCUSSION THREAD

**Discussion by flawrist** [alexhallam/tv#157] [Q&A] [answered]
```console
$ tv sample.csv {magic switch} num_emails
```

**Comment by alexhallam** [comment]
Thanks for the suggestion. I am trying to keep `tv` a viewer only. If you want to do data manipulation on the command line there is nothing better than duckdb cli. https://github.com/alexhallam/tv#use-with-duckdb

```sh
duckdb --csv -c "SELECT passenger_count, COUNT(*), AVG(total_amount) FROM taxi.csv GROUP BY passenger_count ORDER BY passenger_count" | tv 
```

or in your case

```sh
duckdb --csv -c "SELECT * FROM sample.csv ORDER BY num_emails " | tv
```

**Reply by flawrist** [reply]
Unfortunately, your command outputs the data from _num_emails_ column only, omitting the rest.

I ended up with PowerShell:
```powershell
Import-CSV sample.csv | Sort {[int]$_.num_emails} -Desc | Format-Table
```

**Reply by alexhallam** [reply, marked as answer]
Sorry,

here is the correction

```sh
duckdb --csv -c "SELECT * FROM sample.csv ORDER BY num_emails " | tv
```
